app/models/enterprise.rb

```diff
@@ -64,6 +64,7 @@ class Enterprise < ApplicationRecord
   has_many :customers
   has_many :inventory_items
   has_many :tag_rules
+  belongs_to :webhook_endpoint, optional: true, dependent: :destroy
```

Comment:
Would it be better the other way round? One endpoint belongs to an enterprise and an enterprise *can* have one endpoint. It avoids an optional, sparse column on enterprise because the endpoint definitely requires an owner. And we don't want enterprises to share endpoints. It also means that the schema doesn't need changing if we want to change from has_one to has_many.

Reply:
Much better, yes! Updated.